src/foam/dao/CSVSink.js

```diff
@@ -145,7 +257,7 @@ foam.CLASS({
 
   properties: [
     {
-      name: 'toCSV',
+      name: 'javaToCSV',
```

Comment:
Presumably we still want the JS implementations as well, so the client, so  Iwouldn't delete toCSV.

Take a look at src/foam/java/refinements.js for javaCsvParser and/or javaCloneProperty.

You need to add a javaOutputCSVLabel and javaOutputCSVValue or similar properties to Property as class: 'String', update the createJavaPropertyInfo_ method, and then update the methods factory of src/foam/java/PropertyInfo.js